vllm/model_executor/layers/fused_moe/layer.py

```diff
@@ -343,3 +353,186 @@ def _load_fp8_scale(self, param: torch.nn.Parameter,
             # If we are in the row parallel case (down_proj)
             else:
                 param_data[expert_id] = loaded_weight
+
+
+class PhiFusedMoE(torch.nn.Module):
```

Comment:
It looks like this isn't very different from FusedMoE and isn't actually used anywhere. Can we remove this?

Reply:
+1. Seems to have a lot of redundant code.

Reply:
Yes, this hasn't been cleaned up, and it's supposed to be removed.